include/cvc5/c/cvc5.h

```diff
@@ -0,0 +1,3936 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Aina Niemetz
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2023 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * The cvc5 C API.
+ */
+
+#ifndef CVC5__C_API__CVC5_H
+#define CVC5__C_API__CVC5_H
+
+#if __cplusplus
+extern "C" {
+#endif
+
+#define CVC5_API_USE_C_ENUMS
+#include <cvc5/cvc5_kind.h>
+#include <cvc5/cvc5_sort_kind.h>
+#include <cvc5/cvc5_types.h>
+#undef CVC5_API_USE_C_ENUMS
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* -------------------------------------------------------------------------- */
+
+/**
+ * Encapsulation of a three-valued solver result, with explanations.
+ */
+typedef struct Cvc5Result Cvc5Result;
```

Comment:
I guess you're using UpperCamlCase for types here.  Unfortunately, it doesn't play nicely with words that end with numbers.  Especially Cvc5Op looks really weird - looks like the number 50.  What about CvcFive... as a prefix?  Could also use an underscore but that's considered bad in CamlCase.  I looked around and didn't see any better solutions.  Thoughts?

Reply:
Hm, it doesn't look weird to me :grin: `CVC5Op` better?

Reply:
Yeah I don't have a better suggestion.  Let's leave it as is.

Reply:
Is `CvcOp` a possibility?

Reply:
I would rather use the full solver name.